src/itemdef.cpp

```diff
@@ -211,6 +215,8 @@ void ItemDefinition::deSerialize(std::istream &is)
 	inventory_overlay = deSerializeString(is);
 	wield_overlay = deSerializeString(is);
 
+	light_source = readU8(is); // Probably not how I should do this, any help welcome
```

Comment:
follow the instructions of the comment below this

Reply:
How is it now?